includes/core-changes/corefx/1.0/filesysteminfo-attributes-exceptions.md

```diff
@@ -0,0 +1,34 @@
+### UnauthorizedAccessException thrown by FileSystemInfo.Attributes
+
+In .NET Core, an <xref:System.UnauthorizedAccessException> is thrown when the caller attempts to set a file attribute value but doesn't have write permission.
+
+#### Change description
+
+In .NET Framework, an <xref:System.ArgumentException> is thrown when the caller attempts to set a file attribute value in <xref:System.IO.FileSystemInfo.Attributes?displayProperty=nameWithType> but doesn't have write permission. In .NET Core, an <xref:System.UnauthorizedAccessException> is thrown instead.
+
+#### Version introduced
+
+1.0
+
+#### Recommended action
+
+Modify any `catch` statements to catch an <xref:System.UnauthorizedAccessException> instead of, or in addition to, an <xref:System.ArgumentException>, as necessary.
+
+> [!NOTE]
```

Comment:
This note seems to fit better in Change description than in Recommended action, and I don't think it needs to be in a Note box.